hawkmoth/docstring.py

```diff
@@ -99,10 +99,11 @@ def is_doc(comment):
         """Test if comment is a C documentation comment."""
         return comment.startswith('/**') and comment != '/**/'
 
-    def _get_plain_comment(self):
-        """Return plain comment with comment markers and line prefixes removed."""
-        lines = statemachine.string2lines(self._text, 8, convert_whitespace=True)
+    def _get_comment_lines(self):
+        return statemachine.string2lines(self._text, 8, convert_whitespace=True)
 
+    def _remove_comment_markers(self, lines):
+        """Remove comment markers and line prefixes from comment lines."""
```

Comment:
This one can be a static method too. Not that we'll be using it as such I guess.